src/functions.php

```diff
@@ -108,3 +115,51 @@ function cd(string $path): void
         $context->currentDirectory = PathHelper::realpath($context->currentDirectory . '/' . $path);
     }
 }
+
+/** @param (callable(string, string) : (false|null)) $function */
+function watch(string $path, callable $function): void
+{
+    $binary = 'watcher';
+
+    if ('\\' === \DIRECTORY_SEPARATOR) {
+        $binary = 'watcher.exe';
+    }
+
+    $command = [__DIR__ . \DIRECTORY_SEPARATOR . '..' . \DIRECTORY_SEPARATOR . 'watcher' . \DIRECTORY_SEPARATOR . 'bin' . \DIRECTORY_SEPARATOR . $binary, $path];
```

Comment:
Doesn't unix path with `\` works on windows ?

Reply:
Let's use / imho. It's simpler